### ISSUE TITLE
Could not find unf_ext-0.0.7.5 in any of the sources

### ISSUE THREAD

**Issue by MarkAMcCorkle** [fastlane/fastlane#12547]
[17:25:59]: ▸ Bundler::GemNotFound: Could not find unf_ext-0.0.7.5 in any of the sources

This occurs every time fastlane is ran. 

**If I manually run...**
bundle exec pod install
Ignoring unf_ext-0.0.7.5 because its extensions are not built.  Try: gem pristine unf_ext --version 0.0.7.5
Could not find unf_ext-0.0.7.5 in any of the sources
Run `bundle install` to install missing gems.

**If I do what the message says.** 
gem pristine unf_ext --version 0.0.7.5
Restoring gems to pristine condition...
Building native extensions. This could take a while...
Restored unf_ext-0.0.7.5

**The problem persists.**
bundle exec pod install
Ignoring unf_ext-0.0.7.5 because its extensions are not built.  Try: gem pristine unf_ext --version 0.0.7.5
Could not find unf_ext-0.0.7.5 in any of the sources
Run `bundle install` to install missing gems.

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.95.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.2        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.5        |
| xcodeproj                 | 1.5.8        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.6        |
| unicode-display_width     | 1.1.3        |
| terminal-table            | 1.7.3        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| public_suffix             | 2.0.5        |
| tty-screen                | 0.6.4        |
| tty-cursor                | 0.5.0        |
| tty-spinner               | 0.8.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| commander-fastlane        | 4.4.6        |
| faraday                   | 0.12.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| gh_inspector              | 1.1.3        |
| json                      | 1.8.1        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| bundler                   | 1.14.6       |
| faraday_middleware        | 0.11.0.1     |
| naturally                 | 2.1.0        |
| simctl                    | 1.6.4        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| logging                   | 2.2.2        |
| jwt                       | 1.5.6        |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.13.6       |
| emoji_regex               | 0.1.1        |
| highline                  | 1.7.10       |
| excon                     | 0.62.0       |
| CFPropertyList            | 3.0.0        |
| libxml-ruby               | 3.0.0        |
| plist                     | 3.4.0        |
| fastimage                 | 2.1.3        |
</details>

**Comment by joshdholtz** [comment]
@MarkAMcCorkle Heyyy 👋 Have you tried running `bundle update fastlane` or `bundle update` yet to see if that solves this for you?

**Comment by MarkAMcCorkle** [comment]
Yes. No version change.
Bundler attempted to update fastlane but its version stayed the same
fastlane 2.95.0

Also, I get this when checking info on the gem. Could it be a path or env issue?

bundle info unf_ext
  * unf_ext (0.0.7.5)
	Summary: Unicode Normalization Form support library for CRuby
	Homepage: https://github.com/knu/ruby-unf_ext
	Path: /usr/local/opt/gems/gems/unf_ext-0.0.7.5

**Comment by joshdholtz** [comment]
@MarkAMcCorkle Do you only get this when running `bundle exec pod install` but not when running `bundle exec fastlane <lane>`?

**Comment by MarkAMcCorkle** [comment]
**UPDATE:**
I just added `use_bundle_exec: false` to my cocoapods section of Fastfile and now it works fine. Is this the recommended way to handle this now? 


**Earlier response:**
Yes it occurs on both and during the cocoapods install of a lane. 

[20:19:24]: Driving the lane 'ios test' 🚀
[20:19:24]: -----------------------
[20:19:24]: --- Step: cocoapods ---
[20:19:24]: -----------------------
[20:19:24]: $ cd '.' && bundle exec pod install
[20:19:25]: ▸ bundler: failed to load command: pod (/Users/markmccorkle/.fastlane/bin/bundle/bin/pod)
[20:19:25]: ▸ Bundler::GemNotFound: Could not find unf_ext-0.0.7.5 in any of the sources

**Comment by joshdholtz** [comment]
@MarkAMcCorkle That is not the recommended way but it does work in some use cases - and glad that it is working for you now 😊 It sounds like that this maybe a configuration issue with your system. It sounds like there is maybe a conflict between what is specified in your `Gemfile` and what is installed outside of your `Gemfile`.

Do you have `gem 'cocoapods'` in your Gemfile at all?

**Comment by MarkAMcCorkle** [comment]
I started with only fastlane in my gem file then added cocoapods when trying to debug the issue. Adding cocoapods didn't make a difference. I've tried this on 3 different systems now so it very well could be a problem with the project itself rather than the environment. 

Here is my current Gemfile

```
source "https://rubygems.org"
gem "cocoapods"
gem "fastlane"
```

**Comment by joshdholtz** [comment]
@MarkAMcCorkle Yeah, that is _super_ weird. I do this exact same thing all the time and have never run into this issue 🤔 But that `use_bundle_exec: false` is there for this exact reason cause sometimes it happens on some systems 🙃

**Comment by MarkAMcCorkle** [comment]
No worries, as long as I'm building again I can live with the nag screen telling me bundle exec is better, lol. Thanks for your help man! Maybe it'll resolve itself if unf_ext gets a version bump.
